specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2021-10-01/mfe.json

```diff
@@ -0,0 +1,10010 @@
+      "x-ms-enum": {
+        "name": "KeyType",
+        "modelAsString": true
+      },
+      "additionalProperties": false
+    },
+    "ManagedIdentity": {
+      "description": "Managed identity configuration.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/IdentityConfiguration"
+        }
+      ],
+      "properties": {
+        "clientId": {
+          "format": "uuid",
+          "description": "Specifies a user-assigned identity by client ID. For system-assigned, do not set this field.",
+          "type": "string",
+          "x-ms-mutability": [
+            "create",
+            "read"
+          ],
+          "x-nullable": true
+        },
+        "objectId": {
+          "format": "uuid",
+          "description": "Specifies a user-assigned identity by object ID. For system-assigned, do not set this field.",
+          "type": "string",
+          "x-ms-mutability": [
+            "create",
+            "read"
+          ],
+          "x-nullable": true
+        },
+        "resourceId": {
+          "description": "Specifies a user-assigned identity by ARM resource ID. For system-assigned, do not set this field.",
+          "type": "string",
+          "x-ms-mutability": [
+            "create",
+            "read"
+          ],
+          "x-nullable": true
+        }
+      },
+      "x-ms-discriminator-value": "Managed",
+      "additionalProperties": false
+    },
+    "ManagedOnlineDeployment": {
+      "description": "Properties specific to a ManagedOnlineDeployment.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/OnlineDeployment"
+        }
+      ],
+      "properties": {
+        "modelMountPath": {
+          "description": "The path to mount the model in custom container.",
+          "type": "string",
+          "x-nullable": true
+        },
+        "readinessProbe": {
+          "description": "Readiness probe validates if the container is ready to serve traffic. The properties and defaults are the same as liveness probe.",
+          "$ref": "#/definitions/ProbeSettings",
+          "x-nullable": true
+        }
+      },
+      "x-ms-discriminator-value": "Managed",
+      "additionalProperties": false
+    },
+    "MedianStoppingPolicy": {
+      "description": "Defines an early termination policy based on running averages of the primary metric of all runs",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/EarlyTerminationPolicy"
+        }
+      ],
+      "x-ms-discriminator-value": "MedianStopping",
+      "additionalProperties": false
+    },
+    "ModelContainer": {
+      "type": "object",
+      "properties": {
+        "description": {
+          "description": "The asset description text.",
+          "type": "string",
+          "x-nullable": true
+        },
+        "properties": {
+          "description": "The asset property dictionary.",
+          "t
```

Comment:
It's my understanding that "name" is a reserved word and cannot be used as a property name (see https://github.com/Azure/azure-rest-api-specs/pull/12554#issuecomment-812245037). @pilor is SkuSetting.name allowed?

Reply:
SKUs have a standard contract.  https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/Addendum.md#enumerating-skus-for-an-existing-resource